docs/github.md

```diff
@@ -1,737 +1,184 @@
 :::
-<!-- (TODO: Add link to creating a package) -->
-<!-- (TODO: Add link to pushing the package) -->
-
-**Summary**
-
-1. [Go to "Playground"](#go-to-the-playground) > "Manage Projects", create a new Project, and [name it appropriately](name_formats.md#docassemble-projects-and-packages-pascalcase).
-1. Go back to "Playground" and upload your files or make a new file. The name of the YAML file should be the [name of your form](name_formats.md#form-files-stored-in-docassemble-snake_case).
-1. [Go to Packages](#go-to-the-package) and save your package. Its name should be based on the form name and [follow conventions](name_formats.md#docassemble-projects-and-packages-pascalcase).
-1. [Commit/push the package to GitHub](https://docassemble.org/docs/packages.html#github).
-
-**Details**
-
-In the Playground, tap the top left dropdown and select "Manage Projects".
-
-![Navigate to "Manage Projects"](./assets/da_playground_manage_projects.png)
-
-[Make a new Project](https://docassemble.org/docs/playground.html#projects) and [name it appropriately](name_formats.md#docassemble-projects-and-packages-pascalcase).
-
-![Create a new Project](./assets/da_projects_new.png)
-
-![Name a new Project](./assets/da_project_new_project_name.png)
-
-Tap your new Project's name.
-
-![Open a Project](./assets/da_projects_open.png)
-
-The Playground will open the Project with a default "test.yml" file. Either [add a new file](#create-a-new-yaml-file) or upload your generated `.yml` file by tapping "Upload".
-
-![Upload a YAML file in the Playground](./assets/da_playground_upload_yaml.png)
-
-[Create your Package and push it to GitHub](#create-a-new-repository).
-
-### Create a new YAML file
-**Summary**
-
-[Go to the Playground](#go-to-the-playground), tap "Add", and give the file [an appropriate name](name_formats.md#form-files-stored-in-docassemble-snake_case).
-
-**Details**
-
-[Go to the Playground](#go-to-the-playground) and tap "Add".
-
-![Tap "Add"](./assets/da_playground_add.png)
-
-Give the file [an appropriate name](name_formats.md#form-files-stored-in-docassemble-snake_case). Then save the file.
-
-![Name and save the YAML file](./assets/da_playground_save.png)
-
-Your save options should change below the code area to indicate the save was successful.
 
-![The editor buttons change once the file has been saved the first time](./assets/da_playground_after_save.png)
+## Workflow
 
-### Upload a YAML file
-See the [docassemble docs about Playground YAML files](https://docassemble.org/docs/playground.html#interview_files).
+Here is what we recommend for a workflow for using GitHub with the Docassemble playground. Repeat these steps each time you start working on a new task or issue:
 
-### Upload a package
```

Comment:
I think we still need to document this step as it's the first step after using the Weaver. It could link to another part of the documentation though